src/Native/configure.cmake

```diff
@@ -235,10 +249,17 @@ check_cxx_source_runs(
 check_prototype_definition(
     getpriority
     "int getpriority(int which, int who)"
-    "0"
+    0
```

Comment:
@stephentoub quotes are not necessary. All cmake integer parameters can go with or without quotes. I did it for consistency. No harm either way.